target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -272,8 +293,8 @@ TARGET_DEVICES += F5D7633
 ### Broadcom ###
 define Device/BCM96318REF
   $(Device/bcm63xx)
-  IMAGES :=
```

Comment:
If you want to remove empty IMAGES (haven't checked whether this is working), please do it in a separate commit.

Reply:
k, I thought I've read that you wanted to remove them.